translations/base-fr.yaml

```diff
@@ -838,52 +934,55 @@ keybindings:
         mixer: *mixer
         painter: *painter
         trash: *trash
+        storage: *storage
+        wire: *wire
+        constant_signal: *constant_signal
+        logic_gate: Logic Gate
+        lever: Switch (regular)
+        lever_wires: Switch (wires)
+        filter: *filter
+        wire_tunnel: *wire_tunnel
+        display: *display
+        reader: *reader
+        virtual_processor: *virtual_processor
+        transistor: *transistor
+        analyzer: *analyzer
+        comparator: *comparator
+        # ---
 
+        pipette: Pipette
         rotateWhilePlacing: Pivoter
         rotateInverseModifier: >-
             Variante : Pivote à gauche
         cycleBuildingVariants: Alterner entre les variantes
         confirmMassDelete: Confirmer la suppression de la sélection
+        pasteLastBlueprint: Copier le dernier patron
         cycleBuildings: Alterner entre les bâtiments
+        lockBeltDirection: Utiliser le planificateur de convoyeurs
+        switchDirectionLockSide: >-
+            Planificateur : changer de côté
 
         massSelectStart: Cliquez et glissez pour commencer
         massSelectSelectMultiple: Sélectionner plusieurs zones
         massSelectCopy: Copier la sélection
+        massSelectCut: Couper la sélection
 
         placementDisableAutoOrientation: Désactiver l’orientation automatique
         placeMultiple: Rester en mode placement
         placeInverse: Inverser le mode d’orientation automatique
-        pasteLastBlueprint: Copier le dernier patron
-        massSelectCut: Couper la sélection
-        exportScreenshot: Exporter une image de toute la base
-        mapMoveFaster: Se déplacer plus vite
-        lockBeltDirection: Utiliser le planificateur de convoyeurs
-        switchDirectionLockSide: "Planificateur : changer de côté"
-        pipette: Pipette
-        menuClose: Fermer le menu
-        switchLayers: Basculer le calque
-        advanced_processor: Inverseur de couleur
-        energy_generator: Générateur d’énergie
-        wire: Câble
 
 about:
     title: À propos de ce jeu
     body: >-
-        Ce jeu est open source et développé par <a href="https://github.com/tobspr"
-        target="_blank">Tobias Springer</a> (c’est moi).<br><br>
+        Ce jeu est open source et développé par <a href="https://github.com/tobspr" target="_blank">Tobias Springer</a> (c’est moi).<br><br>
```

Comment:
Why did you un-fold lines? It's way more readable with shorter lines.

Reply:
This is to simplify the file comparizon with the english version.
But yes it is true, it is more readable with shorter lines.
Maybe in the english version also the line should be cutted.